pcdsdevices/mpod_apalis.py

```diff
@@ -64,7 +73,7 @@ def set_voltage(self, voltage_number):
         """
         max_voltage = self.max_voltage.get()
 
-        if voltage_number <= max_voltage:
+        if abs(voltage_number) <= abs(max_voltage):
             self.voltage.put(voltage_number)
         else:
             self.voltage.put(max_voltage)
```

Comment:
Self review: I notice now that this fallback case might not be quite what we want.
I will double-check what `max_voltage` looks like for the TMO devices, hopefully it's negative for channels with negative polarity. Otherwise, this will need to be revised.

Reply:
I've confirmed that `max_voltage` is negative for the negative-polarity channels. So this is probably OK but still feels a bit wonky (e.g. how are we supposed to interpret positive numbers put to negative channels?)

Reply:
Should we be bounding the set voltage in the IOC, rather than in python? The DRVH and DRVL fields seem like the right place to do this. But, I'm not familiar with that IOC, so perhaps it's more complicated than that.

Reply:
I have the same opinion, it's much better to do these in the IOC. I am also unfamiliar with the IOC though.

Reply:
I think I'm going to make a jira ticket targeting the IOC feature, referencing this PR. Then, I'm going to complete this PR and ask for reviews.

Reply:
https://jira.slac.stanford.edu/browse/ECS-6495

Reply:
I thought about this some more- maybe this isn't about clamping (maybe the IOC does this already?), but perhaps it's about the case where:
- `max_voltage` is -3000
- call `set_voltage(-100)`
- -100 is not less than -3000
- therefore, put max_voltage of -3000

And with the revised logic:
- `max_voltage` is -3000
- call `set_voltage(-100)`
- 100 is less than 3000
- therefore, put 100

Reply:
It's not immediately obvious from the IOC since the control limit fields are not set, but it's still possible that these are implemented internally in some roundabout way. I'll include a note on the jira.

Reply:
I think what I'll end up doing here is:
- add ctrl limits to the signal (not present in the IOC) to give a clear error when someone tries to put a voltage outside the range to the signal directly
- clamp values in a more straightforward way in `set_voltage` and have this give a warning message
- allow an IOC investigation to be done as future work